web/packages/teleterm/src/mainProcess/resolveNetworkAddress.ts

```diff
@@ -94,11 +94,23 @@ function waitForMatchInStdout(
       }
     };
 
-    const rejectOnError = (error: Error) => {
+    const rejectWithCleanup = (error: Error) => {
       reject(error);
       removeListeners();
     };
 
+    const rejectOnError = (error: Error) => {
+      const errorToReport = error.message.includes(process.spawnfile)
+        ? error
+        : // Attach spawnfile so that the process can be identified without looking at the stacktrace.
+          // resolveNetworkAddress is the only function that ends up surfacing to the UI the error
+          // event of a process.
+          new Error(`${process.spawnfile}: ${error.message}`, {
+            cause: error,
+          });
```

Comment:
I'm not sure if this is necessary, but I don't think there's any guarantee that the error message of an error from [the error event of a process](https://nodejs.org/docs/latest-v22.x/api/child_process.html#event-error) is going to include the spawnfile.

It certainly happens for EACCESS on macOS as shown in the screenshot, hence why I'm prepending the spawnfile only if the message doesn't already include it.

<img width="260" height="332" alt="macos-spawn-error" src="https://github.com/user-attachments/assets/fd2db4a4-d1fb-437c-b44e-dc97187231bd" />